addons/koji/common/src/main/java/org/commonjava/indy/koji/content/KojiBuildAuthority.java

```diff
@@ -153,9 +153,7 @@ public boolean isAuthorized( String path, EventMetadata eventMetadata, ProjectRe
 
             // @formatter:off
             Predicate<KojiArchiveInfo> archiveInfoFilter = ( archive ) -> EXCLUDED_FILE_ENDINGS.stream()
-                                                                                               .filter( ending -> !archive.getFilename().endsWith( ending ) )
-                                                                                               .findAny()
-                                                                                               .isPresent();
+                                                                                               .allMatch( ending -> !archive.getFilename().endsWith( ending ) );
```

Comment:
This is definitely one place where the Streams API hurts clarity. Combined with the stream below, that is a lot harder to understand than the analogous for-loop structures...